ocf_data_sampler/numpy_batch/gsp.py

```diff
@@ -1,20 +1,33 @@
 """Convert GSP to Numpy Batch"""
 
 import xarray as xr
-from ocf_datapipes.batch import BatchKey, NumpyBatch
 
 
-def convert_gsp_to_numpy_batch(da: xr.DataArray, t0_idx: int | None = None) -> NumpyBatch:
+class GSPBatchKey:
+
+    gsp = 'gsp'
+    gsp_nominal_capacity_mwp = 'gsp_nominal_capacity_mwp'
+    gsp_effective_capacity_mwp = 'gsp_effective_capacity_mwp'
+    gsp_time_utc = 'gsp_time_utc'
+    gsp_t0_idx = 'gsp_t0_idx'
+    gsp_solar_azimuth = 'gsp_solar_azimuth'
+    gsp_solar_elevation = 'gsp_solar_elevation'
+    gsp_id = 'gsp_id'
+    gsp_x_osgb = 'gsp_x_osgb'
+    gsp_y_osgb = 'gsp_y_osgb'
+
+
+def convert_gsp_to_numpy_batch(da: xr.DataArray, t0_idx: int | None = None) -> dict:
```

Comment:
If we are removing the `NumpyBatch` and other batch types like the `NWPNumpyBatch` and just calling them dicts should we remove reference to the types as well as well? Like in the docstrings here and other files?